parm/wflow/aqm_prep.yaml

```diff
@@ -69,7 +85,15 @@ metatask_nexus_emission:
 task_nexus_post_split:
   <<: *default_aqm
   command: '&LOAD_MODULES_RUN_TASK; "nexus_post_split" "&HOMEdir;/jobs/JSRW_NEXUS_POST_SPLIT"'
-  join: !cycstr '&LOGDIR;/{{ jobname }}_@Y@m@d@H&LOGEXT;'
+  join:
+    cyclestr:
+      value:  '&LOGDIR;/{{ jobname }}_@Y@m@d@H&LOGEXT;'
+  envars:
+    <<: *default_vars
+    nprocs: '{{ task_nexus_post_split.execution.batchargs.nodes * task_nexus_post_split.execution.batchargs.tasks_per_node }}'
+  nodes: '{{ task_nexus_post_split.execution.batchargs.nodes }}:ppn={{ task_nexus_post_split.execution.batchargs.tasks_per_node }}'
+  exclusive: "True"
```

Comment:
Similar to `plot.yaml`, is there a reason that exclusive is required for `task_nexus_post_split`?  On machines that don't support `exclusive`, this causing warning messages to appear:

`04/09/25 07:59:36 MDT :: FV3LAM_wflow.xml :: Ignoring <exclusive>; this platform does not support it.`